backend/OR-Tools/OR-Tools/ORSolver.cs

```diff
@@ -102,6 +104,12 @@ public void PenaltiesAndDroppingVisits()
 
         public void PrintSolution()
         {
+            if (solution == null)
+            {
+                System.Console.WriteLine("No solution");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4618867)